spyder/plugins/ipythonconsole/widgets/client.py

```diff
@@ -135,6 +136,7 @@ def __init__(self, plugin, id_,
         self.allow_rename = True
         self.stderr_dir = None
         self.is_error_shown = False
+        self.css_path = css_path

```

Comment:
Let's change this to

```python
if css_path is None:
    self.css_path = CSS_PATH
else:
    self.css_path = css_path
```

This way tests for the IPython console alone will pass without problems.